### DISCUSSION TITLE
Flip Project Controls Location Addon

### DISCUSSION THREAD

**Discussion by DatGamerYoloYT** [ScratchAddons/ScratchAddons#4491] [Ideas] [answered]
Ok so basically I want it kinda like this:
![image](https://user-images.githubusercontent.com/84749503/163047789-4f3bc63d-d85a-41d0-8d96-7c57b8edf38a.png)
Because it's kinda like this right now:
![image](https://user-images.githubusercontent.com/84749503/163047906-ed12e6e8-a555-4ce2-bb9c-56c010b50864.png)

if something like this doesn't get added then i guess it'll be impossible to recreate scratch 2.0 in 3.0

**Comment by Samq64** [comment, marked as answer]
#2010

**Reply by DatGamerYoloYT** [reply]
oh
